lib/text-formatters.js

```diff
@@ -45,3 +45,12 @@ function metric(n) {
   return ''+n;
 }
 exports.metric = metric;
+
+// Remove the starting v in a string.
+function omitv(version) {
+  if (version.charCodeAt(0) === 118) {
+    return version.slice(1);
+  }
+  return version;
+}
```

Comment:
There is a duplicate implementation of this function in `lib/php-version.js`. This seems like a better place for it to live. Could you remove that one?

Reply:
Okay, I've replaced `lib/php-version.js` `omitv` function declaration with import from `lib/text-formatters.js`